app/models/concerns/roleable.rb

```diff
@@ -20,7 +20,7 @@ def regular?
 
   def grant_moderation!
     set_permission('moderator', true)
-    enqueue_welcome_moderator_message
+    enqueue_staff_welcome_message('moderator')
```

Comment:
Why the choice of strings over symbols here? `:moderator`

Reply:
No good reason. I updated it with symbols now.